src/main/java/uk/gov/mca/beacons/api/controllers/NoteController.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.mca.beacons.api.controllers;
+
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.UUID;
+import javax.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.mca.beacons.api.domain.Note;
+import uk.gov.mca.beacons.api.domain.User;
+import uk.gov.mca.beacons.api.dto.NoteDTO;
+import uk.gov.mca.beacons.api.dto.WrapperDTO;
+import uk.gov.mca.beacons.api.mappers.NoteMapper;
+import uk.gov.mca.beacons.api.services.GetUserService;
+import uk.gov.mca.beacons.api.services.NoteService;
+
+@RestController
+@RequestMapping("/note")
+@Tag(name = "Note Controller")
+public class NoteController {
+
+  private final NoteMapper noteMapper;
+  private final NoteService noteService;
+  private final GetUserService getUserService;
+
+  @Autowired
+  public NoteController(
+    NoteMapper noteMapper,
+    NoteService noteService,
+    GetUserService getUserService
+  ) {
+    this.noteMapper = noteMapper;
+    this.noteService = noteService;
+    this.getUserService = getUserService;
+  }
+
+  @PostMapping
+  @ResponseStatus(HttpStatus.CREATED)
+  public WrapperDTO<NoteDTO> createNote(
+    @RequestBody @Valid WrapperDTO<NoteDTO> dto
+  ) {
+    final Note note = noteMapper.fromDTO(dto.getData());
+    if (userDoesNotExist(note)) {
+      // TODO: make the id passed in (B2C auth Id) not just null once we're getting that!
+      final User user = getUserService.getUser(null);
+
+      note.setPersonId(UUID.fromString(user.getAuthId()));
```

Comment:
this doesn't feel right, the authId is effectively the azureId, and we have an entity called Person which is unrelated to azure. PersonId suggestions the beacon-db id of the person entity, but authId sugests the azureId?

Reply:
Yeah I totally get what you mean, wondering what the best term would be - but I'm struggling 😅  
Cause it is the `azureId` but `authId` on the `Note` doesn't feel right, cause it's not the note's id...

Would `userAuthId` instead of `personId` on `Note`s sound a bit better? 🤔